tssc/utils/maven.py

```diff
@@ -0,0 +1,128 @@
+"""
+Shared utils for maven deeds
+"""
+
+import xml.etree.ElementTree as ET
+
+def generate_maven_settings(working_dir, maven_servers, maven_repositories, maven_mirrors):
+
```

Comment:
@ppremru @akim309 exrra whitespace here

Reply:
done

Reply:
@akim309 - can you please add comments to the helper methods (eg: parameters, return values)